rllab/envs/box2d/cartpole_env.py

```diff
@@ -17,24 +16,21 @@ def __init__(self, *args, **kwargs):
         self.max_pole_speed = 4.
         self.reset_range = 0.05
         super(CartpoleEnv, self).__init__(
-            self.model_path("cartpole.xml.mako"),
-            *args, **kwargs
-        )
+            self.model_path("cartpole.xml.mako"), *args, **kwargs)
         self.cart = find_body(self.world, "cart")
         self.pole = find_body(self.world, "pole")
+        self.spec = super().spec()
```

Comment:
If you are going to call a superclass, the right way to do it is
```python
super(Box2DEnv, self).spec()
```
so that the correct superclass is chosen (this class has two).

...but storing this it kind of an odd thing to do. Why not just make spec an property in `Box2DEnv` and then let polymorphism do the job?